packages/forma-36-website/src/components/ComponentSource.js

```diff
@@ -37,7 +37,7 @@ class ComponentSource extends React.Component {
   render() {
     return (
       <div className="f36-margin-bottom--m">
-        <LiveProvider code={this.props.children} scope={{ ...f36Components, Grid, GridItem, Autocomplete }}>
+        <LiveProvider code={this.props.children.trim()} scope={{ ...f36Components, Grid, GridItem, Autocomplete, Flex, ProductIcon }}>
```

Comment:
Currently, when we have anything in the examples that is declared or imported, we are getting errors from `LiveProvider`. Does anyone have an idea on how to solve this issue?